helloworld.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```

Comment:
per the [conversation with Tom](https://github.com/TomWhitwell/SlowMovie/pull/29#discussion_r606103910) the idea of this file is to be kind of a minimal "proof of concept" for people to see how the system worked. If that's the case maybe we could strip this __way__ down. Basically hardcode the test file and pull random frames from it. Less code to maintain as changes happen going forward. Could do this in a new PR after the merge.

Reply:
This file currently does something slowmovie.py doesn't: play random frames from the whole folder. That can probably be added, and this one stripped down to serve as example code.